packages/server/controllers/auth.js

```diff
@@ -13,6 +14,9 @@ async function create(accessToken, refreshToken, profile, callback) {
     let { sub, given_name, family_name, picture, email } = profile._json;
     picture = picture.replace("=s96-c", "");
     const googleUUID = uuidv5(sub, NAMESPACE);
+    const hash = crypto.createHash("sha256");
+    hash.update(email);
+    const emailHash = hash.digest("hex");
```

Comment:
It would be better to extract this code into a function that can be used whenever needed instead of copy-pasting duplicate code whenever we want to hash an email. Move this into a `hashUserEmail()` function like in the Users controller and substitute the places you're currently hashing emails to call this new function instead.